import-automation/executor/config_override.json

```diff
@@ -1,8 +1,7 @@
 {
     "configs":{
-        "executor_type": "CLOUD_RUN",
+        "executor_type": "CLOUD_BATCH",
```

Comment:
Are we changing the default to CLOUD_BATCH for all jobs or should we test a bit more to understand logging/monitoring before this?